web/api/report_server.thrift

```diff
@@ -247,6 +247,16 @@ struct BugPathLengthRange {
   2: i64  max, // Maximum value of bug path length.
 }
 
+struct DateInterval {
+  1: i64  before, // Date timestamp in UTC format.
```

Comment:
UTC format is not an integer it is something like this `2010-11-12T13:14:15Z`, it this the unix (epoch) time?

Reply:
Yes, you are right. I fixed it.